src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/CustomCurrencyCodeConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Linq;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public class CustomCurrencyCodeConverter
+
+    {
+        private readonly Type[] types;
+
+
+        public CustomCurrencyCodeConverter(params Type[] types)
+        {
+            this.types = types;
+        }
+
+
+        public CustomCurrencyCodeConverter()
+        {
+        }
+
+
+        public bool CanRead => true;
+
+
+        public bool CanConvert(Type objectType)
+        {
+            return this.types.Any(t => t == objectType);
+        }
+
+
+        //public object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        //{

```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)